runner/ansible/vars/gcp/10-default.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+expected_core:
+  "1.1.1": "20000"
+  "1.1.2": "" # unkown value, default may apply
```

Comment:
Typo `unkown` -> `unknown`.
Like in https://github.com/trento-project/trento/pull/825 I would remove the `default may apply` comment or add some real value (no empty)

Reply:
Fixed